extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_iframe_transport.html

```diff
@@ -41,6 +37,37 @@
         }
       </script>
     </amp-analytics>
+    <amp-ad-exit id='exit-api'>
+      <script type='application/json'>
+        {
+          "targets": {
+            "target1": {
+              "finalUrl": "/?product1&x=CLICK_X&y=CLICK_Y&e=_elem&shouldNotBeReplaced=AMP_VERSION",
+              "vars": {
```

Comment:
what is this file used for? Should it demonstrate the new variable substitution?